src/pages/Home/Home.tsx

```diff
@@ -23,17 +23,17 @@ const Home: React.FC = () => {
   const [salaries, setSalaries] = useState([
     {
       company: "Company Name Here",
```

Comment:
I think we have different linters, probably would be nice to setup Pretier or something like that tin order to avoid these type of changes.